src/EditorFeatures/Core/LanguageServer/Handlers/Symbols/DocumentSymbolsHandler.cs

```diff
@@ -53,7 +53,8 @@ public async Task<object[]> HandleRequestAsync(DocumentSymbolParams request, Req
             // TODO - Return more than 2 levels of symbols.
             // https://github.com/dotnet/roslyn/projects/45#card-20033869
             using var _ = ArrayBuilder<object>.GetInstance(out var symbols);
-            if (context.ClientCapabilities?.TextDocument?.DocumentSymbol?.HierarchicalDocumentSymbolSupport == true)
+            if (context.ClientCapabilities?.TextDocument?.DocumentSymbol?.HierarchicalDocumentSymbolSupport == true
+                || context.ServerKind == WellKnownLspServerKinds.AlwaysActiveVSLspServer)
```

Comment:
This appears to break [TestGetDocumentSymbolsAsync__WithoutHierarchicalSupport](https://github.com/dotnet/roslyn/blob/565f8b4073b821d9d1d85859f8b34908f5a3d17d/src/Features/LanguageServer/ProtocolUnitTests/Symbols/DocumentSymbolsTests.cs#L51). @dibarbet what is the correct way to resolve this?

Reply:
The solution David mentioned earlier should fix it: https://github.com/dotnet/roslyn/pull/61784#discussion_r892978133. Since it's now blocking the PR I'll add it